docs/framework/wcf/feature-details/web-services-protocols-interoperability-guide.md

```diff
@@ -1,107 +1,109 @@
-  
- Infrastructure protocols interoperability is provided by the WS-* specifications. WCF channels provide support for a number of WS-\* infrastructure protocols. WCF channels are configured using binding elements. The following tables contain the full list of the WS-\* infrastructure protocols implemented by various WCF binding elements.  
-  
- <xref:System.ServiceModel.Channels.HttpTransportBindingElement> supports the specifications in the following table.  
-  
-|Specification/document|Link|  
-|-----------------------------|----------|  
-|HTTP 1.1|[RFC 2616](https://go.microsoft.com/fwlink/?LinkId=90372)|  
-|SOAP 1.1 HTTP Binding|[Simple Object Access Protocol (SOAP) 1.1](https://go.microsoft.com/fwlink/?LinkId=90520), Section 7|  
-|SOAP 1.2 HTTP Binding|[SOAP Version 1.2 Part 2: Adjuncts (Second Edition)](https://go.microsoft.com/fwlink/?LinkId=95329), Section 7|  
-  
- <xref:System.ServiceModel.Channels.TextMessageEncodingBindingElement> and <xref:System.ServiceModel.Channels.MtomMessageEncodingBindingElement> support the specifications in the following table.  
-  
-|Specification/Document|Link|  
-|-----------------------------|----------|  
-|XML|[Extensible Markup Language (XML) 1.0 (Fourth Edition)](https://go.microsoft.com/fwlink/?LinkId=15139)|  
-|SOAP 1.1|[Simple Object Access Protocol (SOAP) 1.1](https://go.microsoft.com/fwlink/?LinkId=96687)|  
-|SOAP 1.2 Core|[SOAP Version 1.2 Part 1: Messaging Framework (Second Edition)](https://go.microsoft.com/fwlink/?LinkId=94664)|  
-|WS-Addressing 2004/08|[Web Services Addressing (WS-Addressing)](https://go.microsoft.com/fwlink/?LinkId=81239)|  
-|W3C Web Services Addressing 1.0 - Core|[Web Services Addressing 1.0 - Core](https://go.microsoft.com/fwlink/?LinkId=96688)|  
-|W3C Web Services Addressing 1.0 - SOAP Binding|[Web Services Addressing 1.0 - SOAP Binding](https://go.microsoft.com/fwlink/?LinkId=96689)|  
-|W3C Web Services Addressing 1.0 - WSDL Binding*|[Web Services Addressing 1.0 - WSDL Binding](https://go.microsoft.com/fwlink/?LinkId=96690)|  
-|W3C Web Services Addressing 1.0 Metadata|[Web Services Addressing 1.0 - Metadata](https://www.w3.org/TR/ws-addr-metadata/)|  
-|WSDL SOAP1.1 Binding|[Web Services Description Language (WSDL) 1.1](https://go.microsoft.com/fwlink/?LinkId=96160)|  
-|WSDL SOAP1.2 Binding|[WSDL 1.1 Binding Extension for SOAP 1.2](https://go.microsoft.com/fwlink/?LinkId=96691)|  
-  
- <xref:System.ServiceModel.Channels.MtomMessageEncodingBindingElement> supports the specifications in the following table.  
-  
-|Specification/document|Link|  
-|-----------------------------|----------|  
-|XOP|[XML-binary Optimized Packaging](https://go.microsoft.com/fwlink/?LinkId=96714)|  
-|MTOM + SOAP1.2 Binding|[SOAP Message Transmission Optimization Mechanism](https://go.microsoft.com/fwlink/?LinkId=96713)|  
-|MTOM SOAP 1.1 Binding|[SOAP 1.1 Binding for MTOM 1.0](https://go.microsoft.com/fwlink/?LinkId=96712)|  
-|MTOM WS-PolicyAssertions|To be published.|  
```

Comment:
I've added a link to [MTOM Serialization Policy Assertion (WS-MTOMPolicy)](https://www.w3.org/Submission/WS-MTOMPolicy/) instead of "To be published.".